### PR TITLE
chevron bug on About page #1481

### LINKED ISSUE #1481: Fix arrow bug on About page
### Overview
There seems to be a problem with how the card's accordion arrows are oriented when the page is resized from desktop to mobile. The accordion arrows seem to point the wrong way, but fix themselves as the accordion is closed and reopened. See the gif below for more details.

<details>
<summary>Gif</summary>

about_page_accordion
</details>

### Action Items
- [ ] Ensure that the accordion arrow points the right way when you change from desktop to mobile view

### Resources/Instructions
About Page: https://github.com/hackforla/website/blob/gh-pages/pages/about.html
About SCSS file: https://github.com/hackforla/website/blob/gh-pages/_sass/components/_about.scss
About JS file: https://github.com/hackforla/website/blob/gh-pages/assets/js/about.js

### PR DESCRIPTION
**Pull request by josiehandeveloper**
https://user-images.githubusercontent.com/63170710/117094818-ac36eb00-ad19-11eb-9da3-0bb6287819b7.mov

fixes #1481

### CODE REVIEW

File: assets/js/about.js [hunk 1]
```diff
@@ -57,12 +57,16 @@ function changeDisplayMode() {
   if (window.innerWidth < BREAK_POINT && FLAGS.displayChanged) {
     document.querySelectorAll(".page-card--about:nth-child(n+3)").forEach((pageCard) => {pageCard.lastElementChild.style.display = "none"; });
     document.querySelector(`div[data-hash="${window.location.hash.split("#")[1]}"]`).parentElement.lastElementChild.style.display = "block";
+    document.querySelector(`div[data-hash="${window.location.hash.split("#")[1]}"]`).parentElement.children[0].classList.toggle("au_active")
+
     FLAGS.displayChanged = false;
+    console.log("window width less")
```
Comment (akibrhast) [inline assets/js/about.js]: remove console log please

File: assets/js/about.js [hunk 2]
```diff
@@ -57,12 +57,16 @@ function changeDisplayMode() {
...
   } 
   else if (window.innerWidth > BREAK_POINT && !FLAGS.displayChanged) {
     document.querySelectorAll(".page-card--about:nth-child(n+3)").forEach((pageCard) => { pageCard.lastElementChild.style.display = "block"; });
-    document.querySelector(".au_active") &&  document.querySelector(".au_active").classList.toggle("au_active");
+    document.querySelectorAll(".au_active").forEach((pageCard) => { pageCard.classList.remove("au_active"); });
     FLAGS.displayChanged = true;
+    console.log("window width greater");
```
Comment (akibrhast) [inline assets/js/about.js]: remove console log please

File: assets/js/about.js [hunk 3]
```diff
@@ -92,23 +96,45 @@ function setupAccordionEventLitseners() {
...
+//     document.querySelectorAll(".page-card--about:nth-child(n+3)").forEach((pageCard) => {
+//         pageCard.firstElementChild.classList.remove("au_active");
+//         pageCard.lastElementChild.style.display = "none";
+//     });
+//     event.target.classList.toggle("au_active");
+//     this.parentElement.lastElementChild.style.display = "block";
+//     window.location.hash = event.target.dataset.hash;
+//   }
+// }
+
```
Comment (akibrhast) [inline assets/js/about.js]: If `new code` works please delete `complicated old code` from the file instead of just commenting it out

**Review by akibrhast — CHANGES_REQUESTED** [review]

**Comment by josiehandeveloper** [comment]
Screen Shot 2021-05-05 at 2 43 09 PM (2)
console.logs removed
Screen Shot 2021-05-05 at 2 44 21 PM (2)
Commented code deleted

**Review by akibrhast — APPROVED** [review]
lgtm. Nice work on refactoring out old complicated code! 👍

**Pull request merged**